src/data/battle_move_effects.h

```diff
@@ -1114,6 +1108,12 @@ const struct BattleMoveEffect gBattleMoveEffects[NUM_BATTLE_MOVE_EFFECTS] =
         .encourageEncore = TRUE,
     },
 
+    [EFFECT_PLACEHOLDER] =
+    {
+        .battleScript = BattleScript_EffectPlaceholder,
+        .battleTvScore = 0, // TODO: Assign points
+    },
+
```

Comment:
Why move this back?

Reply:
Idk how this happened. Seeing this change for the first time. I looked through the commit history and it looks like the change appeared after I merged upcoming. I'll fix it tomorrow. 

> I was about to say approve, but found one more thing, sorry D:

all good

Reply:
If you want you can do a suggested change and I'll accept it

Reply:
Fixed